addons/mail/static/src/messaging/component/chatter_topbar/chatter_topbar.xml

```diff
@@ -4,53 +4,60 @@
+                            Schedule activity
+                        </button>
                     </t>
-                </button>
-                <div class="o-autogrow"/>
-                <t t-if="chatter.hasFollowers">
-                    <div class="o_ChatterTopbar_rightSection">
-                        <div class="o_ChatterTopbar_followers">
-                            <button class="btn btn-link o_ChatterTopbar_buttonFollow" type="button" t-att-disabled="chatter.isDisabled" t-on-click="_onClickFollow">
-                                Follow
-                            </button>
-                            <button class="btn btn-link o_ChatterTopbar_buttonFollowers" type="button" t-att-disabled="chatter.isDisabled" t-on-click="_onClickFollowers">
-                                <i class="fa fa-user"/>
-                                <span class="o_ChatterTopbar_buttonCount o_ChatterTopbar_buttonFollowersCount pl-1">0</span>
-                            </button>
+                    <button class="btn btn-link o_ChatterTopbar_buttonAttachments" type="button" t-att-disabled="chatter.isDisabled" t-on-click="_onClickAttachments">
+                        <i class="fa fa-paperclip"/>
+                        <t t-if="chatter.isDisabled or chatter.thread and chatter.thread.areAttachmentsLoaded">
+                            <span class="o_ChatterTopbar_buttonCount o_ChatterTopbar_buttonAttachmentsCount" t-esc="chatter.thread ? chatter.thread.allAttachments.length : 0"/>
+                        </t>
+                        <t t-else="">
+                            <i class="o_ChatterTopbar_buttonAttachmentsCountLoader fa fa-spinner fa-spin"/>
+                        </t>
+                    </button>
+                    <div class="o-autogrow"/>
+                    <t t-if="chatter.hasFollowers">
+                        <div class="o_ChatterTopbar_rightSection">
+                                <div class="o_ChatterTopbar_followers">
+                                    <button class="btn btn-link o_ChatterTopbar_buttonFollow" type="button" t-att-disabled="chatter.isDisabled" t-on-click="_onClickFollow">
+                                        Follow
+                                    </button>
+                                    <button class="btn btn-link o_ChatterTopbar_buttonFollowers" type="button" t-att-disabled="chatter.isDisabled" t-on-click="_onClickFollowers">
+                                        <i class="fa fa-user"/>
+                                        <span class="o_ChatterTopbar_buttonCount o_ChatterTopbar_buttonFollowersCount pl-1">0</span>
+                                    </button>
+                                </div>
                         </div>
+                    </t>
+                </div>
+                <t t-if="props.hasCloseButton">
+                    <div class="o_ChatterTopbar_buttonClose">
+                        <i class="fa fa-times"/>
                     </div>
```

Comment:
probably missing title and/or aria stuff (see reference on the other comments I added on another PR, don't remember which)

Reply:
?

Reply:
?? (please clarify what is not clear)

Reply:
I don't see what you're talking about in fact ... Maybe it was not a PR of mine ?

Reply:
Basically `aria-hidden="true"` on `<i>` and `title="..."` (and, if different, `aria-label="..."`) on `<div>`

Reply:
https://github.com/odoo-dev/odoo/pull/203#discussion_r404781882

Reply:
Ah didn't read your review as AKU seemed to take care of it